console/console-init/ui/src/modules/iot-device/containers/DeviceListContainer/DeviceListContainer.tsx

```diff
@@ -213,7 +184,9 @@ export const DeviceListContainer: React.FC<IDeviceListContainerProps> = ({
       {
         title: enabled ? "Disable" : "Enable",
         onClick: () =>
```

Comment:
it can be more simple like 
onClick:()=> toggleDeviceStatus(rowData, enabled?false:true)

Reply:
It had occured to me! But it seems bit weird to pass arguments conditionally as it has not been so anywhere else in the codebase.